src/client/user/user-list.js

```diff
@@ -22,3 +24,24 @@ export default function UserList(props) {
         </div>
     );
 }
+
+function showScore(score){
+    if(score !== undefined){
+        return (
+            <span className='badge'>{score}</span>
+        );
+    }
+    return null
+}
+
+function sort(users) {
+    users.sort((a, b) => {
+        let score = 0-(a.data.score-b.data.score);
+        if(a.score !== undefined && b.data.score !== undefined && score !== 0)
```

Comment:
Probably should stick with brackets. In this case you only need the if part and then the else can be outside the if block